collectors/charts.d.plugin/nut/README.md

```diff
@@ -1,6 +1,6 @@
-# nut
+# NUT monitoring with Netdata
```

Comment:
NUT is a suite of tool used by the module to get UPS statistics

> https://en.wikipedia.org/wiki/Network_UPS_Tools

Module monitors UPS

Reply:
**Network UPS tools monitoring with Netdata**